src/components/directory-content.js

```diff
@@ -383,6 +379,10 @@ const DirectoryContent = () => {
                                         event.rowData.accessRights.private
                                     );
                                 }
+                                setMousePosition({
```

Comment:
16 and 4 values should be saved as a constants and should be shared with the same value of the other PR (root folder menu)